staging/src/k8s.io/client-go/tools/clientcmd/api/helpers_test.go

```diff
@@ -240,8 +241,8 @@ func Example_minifyAndShorten() {
 	// users:
 	//   red-user:
 	//     LocationOfOrigin: ""
-	//     client-certificate-data: REDACTED
-	//     client-key-data: REDACTED
+	//     client-certificate-data: DATA+OMITTED
+	//     client-key-data: DATA+OMITTED
```

Comment:
did this really change?

Reply:
and why

Reply:
This test is for the ShortenConfig() function which no longer handles the process of redacting secrets and instead only omits byte data.